superset-frontend/packages/superset-ui-chart-controls/src/shared-controls/customControls.tsx

```diff
@@ -41,6 +41,53 @@ import {
 import { checkColumnType } from '../utils/checkColumnType';
 import { isSortable } from '../utils/isSortable';
 
+// Aggregation choices with computation methods for plugins and controls
+export const aggregationChoices = {
+  raw: {
+    label: 'Force server-side aggregation',
```

Comment:
I'm not sure if this is clearer to the user... "server-side" is an implementation detail.

My understanding is that when this option is selected we ignore the time grain when computing the big number. If the grain is monthly, and the metric is `AVG(price)`, we would have:

- The "trendline" (which is not a trendline) would show a timeseries of `AVG(price)` per month.
- The big number would show the overall `AVG(price)`, from start to end of the data.

So I think renaming this to "Overall value" would make it clearer.

Reply:
I would strongly prefer something else here, since the user shouldn't care if the aggregation is done on the server or on the client. In fact, one might argue that if the chart is slow a user might want to choose this "to make things faster since it forces the aggregation to run on the server".

A few suggestions:

- "Overall" or "overall value"
- "Whole time period"
- "Whole series"

Reply:
I think "Overall value" is the clearest label to me